WcaOnRails/spec/controllers/api_competitions_controller_spec.rb

```diff
@@ -267,6 +275,54 @@
         expect(parsed_body).to eq(
           "formatVersion" => "1.0",
           "id" => "TestComp2014",
+          "events" => [
+            {
+              "id" => "333",
+              "rounds" => [
+                {
+                  "id" => "333-1",
+                  "format" => "a",
+                  "timeLimit" => {
+                    "centiseconds" => TimeLimit::TEN_MINUTES_IN_CENTISECONDS,
+                    "cumulative" => false,
```

Comment:
Strange, I thought I renamed this from cumulative to cumulative across round ids or something... When I get home, I'll check if I failed to push my latest changes.

Empty array sounds reasonable here, I can make that change.